quizs/Jk/jk-quiz.css

```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=M+PLUS+Code+Latin&family=Offside&display=swap');
+  
+body {  
+    margin: 0px;
+    padding: 0px;
+    color: var(--white);
+    background-color: black;
+    font-size: 16px;
+}
+.logo{
+    padding: 2rem 0px 2rem 4rem;
+}
+:root {
+    --red: #9e0101;
+    --dark-red: #612626;
+    --white: white;
```

Comment:
we can be consistent while using colors and use only hex or only RGB or only names(not recommended)

Reply:
okay sure will look into that 
and submit changes as soon as possible